teamlist.md

```diff
@@ -0,0 +1,54 @@
+Teams
+-----
+
+- **Admin**:
+Admin team for the self hosted coala servers and ci.
+
+- **Bears**:
+Developing and maintaining the coala-bears repository. Wrap more
+existing tools, help with writing new bears and update existing ones.
+
+- **cib**:
+The cib team develops and maintains the cib tool for coala.
+
+- **Community**:
+The community team organizes community bonding events like meetups and
+gaming evenings to support and promote a healthy, happy community.
+It also serves as a contact for personal problems or conflicts between
+community members.
+
+- **Core**:
+Developing and maintaining the coala core repository.
+In addition everything that does not fit into another team's domain,
+will be assigned to the core team.
+
+- **Design**;
+Creating mockups and designs for things like a homepage, gui and logos
+for posters, sirts and flyers.
+
+- **Docs**:
+Improve the existing documentation, wiki, tutorials and guides.
+Introduce new ones as needed.
+
+- **Engagement**:
+Create all promotional material like flyers, blog posts,homepage
+content, social media presence and such. Also manage conference
+attendance and feedback handling.
+
+- **Internships**:
+Temporary admin team for things like the Google Summer of Code that
+don't need to be done the whole year.
+
+- **Plugins**:
+Develop and maintain the coala plugins for multiple text editors and
+IDEs.
+
+- **Research**:
+Work on new architecture for coala parts. Help with thesis writing and
+implementing algorithms for the science community.
+
+- **Review**:
+If there is a big PR or a very complicated one this team is meant to
+come from outside the other teams to help with the review process.
+Also in general this team should train the rest of the org how to do
+good reviews.
```

Comment:
I'm about to remove all teams without an application just so this gets merged :D 
Will remove.